### DISCUSSION TITLE
Please update information regarding Galaxy Brain Badge in Community Discussions

### DISCUSSION THREAD

**Discussion by ebndev** [Puliczek/github-achievements-academy#626] [Galaxy Brain - Q&A]
Hi, Community Moderator on GitHub Discussions here!

We would really appreciate if you updated your website regarding the Galaxy Brain Badge, as it is no longer awarded in Community Discussions. However, users are still able to get the badge in other Discussions forums, such as this one.

For any questions, please see more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Thank you!

**Comment by Pix3lexe** [comment]
Yes that would be great. I was really confused why I didn't earn the achievement after answering Community Discussions.

**Comment by mygu** [comment]
Can Galaxy Brain Badge still be obtained by answering now?

**Reply by iamriskycahyan** [reply]
sure thing

**Comment by Simonmatharesh** [comment]
Find an unanswered question and answer the question, remember to answer correctly! Also your answer must be selected as the accepted answer by owner of the question
Reach 2 accepted answers in any category

**Comment by Puliczek** [comment]
Fixed! Sorry it took forever, guys... :(

**Reply by ebndev** [reply]
Thank you so much! 🙏
